streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -96,24 +99,7 @@
             final String group = "stream-task-metrics";
 
             // first add the global operation metrics if not yet, with the global tags only
-            final Map<String, String> allTagMap = metrics.tagMap("task-id", "all");
-            final Sensor parent = metrics.threadLevelSensor("commit", Sensor.RecordingLevel.DEBUG);
-            parent.add(
-                new MetricName("commit-latency-avg", group, "The average latency of commit operation.", allTagMap),
-                new Avg()
-            );
-            parent.add(
-                new MetricName("commit-latency-max", group, "The max latency of commit operation.", allTagMap),
-                new Max()
-            );
-            parent.add(
-                new MetricName("commit-rate", group, "The average number of occurrence of commit operation per second.", allTagMap),
-                new Rate(TimeUnit.SECONDS, new Count())
-            );
-            parent.add(
-                new MetricName("commit-total", group, "The total number of occurrence of commit operations.", allTagMap),
-                new CumulativeCount()
-            );
+            final Sensor parent = commitOverTasksSensor(metrics);
```

Comment:
Do you mind qualifying this reference as:
```suggestion
            final Sensor parent = ThreadMetrics.commitOverTasksSensor(metrics);
```

I had a slightly confusing time confirming that this is still a thread-level sensor, as desired.

Reply:
Actually, it might be a good idea to qualify all references to these static sensor methods, just to keep their associations straight while reading the code.

Reply:
Agreed